android/src/main/java/com/onegini/mobile/sdk/reactnative/clean/use_cases/RegisterUserUseCase.kt

```diff
@@ -14,7 +14,7 @@ import com.onegini.mobile.sdk.reactnative.mapers.UserProfileMapper
 
 class RegisterUserUseCase(private val oneginiSDK: OneginiSDK) {
 
-    operator fun invoke(identityProviderId: String?, scopes: ReadableArray, promise: Promise) {
+    operator fun invoke(identityProviderId: String?, scopes: ReadableArray?, promise: Promise) {
```

Comment:
Shouldn't it have non null params?

Reply:
I think these can all be null. There is a default for the scopes in the android SDK if an empty array is passed in. And the default identityProvider is used when null is passed in there. 
`  public void registerUser(@Nullable OneginiIdentityProvider identityProvider, @Nullable String[] scopes, @NonNull OneginiRegistrationHandler registrationHandler) {
`

Reply:
ah yes you're right